app/views/application/_navbar.html.haml

```diff
@@ -0,0 +1,24 @@
+%nav.navbar.navbar-inverse
+  .container-fluid
+    .navbar-header
+      %button.navbar-toggle.collapsed{"aria-controls" => "navbar", "aria-expanded" => "false", "data-target" => "#navbar", "data-toggle" => "collapse", :type => "button"}
```

Comment:
I have fixed it now. Identical to this line without seeing this line. I get it now, finally. 
(doesnt mean I wont make the misstake again, but now I understand what the difference is.